.github/workflows/ci-workflow.yml

```diff
@@ -97,11 +96,11 @@ jobs:
             ~/.m2/repository
           # Cache Maven modules using a cache key different from setup-java steps
           key: ${{ runner.os }}-maven-static-analysis-${{ hashFiles('**/pom.xml') }}
-      - name: Set up Java 17
+      - name: Set up Java 21
         uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
-          java-version: '17'
+          java-version: '21-ea'
```

Comment:
We should avoid switching an Early Access build version and wait until the official release version is ready.